src/internal/utils/node-belongs.ts

```diff
@@ -15,7 +15,10 @@ export function nodeBelongs(container: Node | null, target: Node | EventTarget |
   if (!(target instanceof Node)) {
     return false;
   }
-  const portal = findUpUntil(target as HTMLElement, node => !!node.dataset.awsuiReferrerId);
+  const portal = findUpUntil(
+    target as HTMLElement,
+    node => node instanceof HTMLElement && !!node.dataset.awsuiReferrerId
```

Comment:
The fix is to restore this line from the past revision: https://github.com/cloudscape-design/components/commit/b28e7f5e83d747a6a21c91150796eb57cdb1db20#diff-eb750a117dea1d7137bc6b95a880c6685b362f28af8d6202fe481e5cccefba27L16

I assumed this check useless, but apparently it is useful